pkg/cloudprovider/providers/azure/azure_blobDiskController.go

```diff
@@ -91,7 +91,7 @@ func (c *BlobDiskController) CreateVolume(name, storageAccount, storageAccountTy
 		accounts = append(accounts, accountWithLocation{Name: storageAccount})
 	} else {
 		// find a storage account
-		accounts, err = c.common.cloud.getStorageAccounts()
+		accounts, err = c.common.cloud.getStorageAccounts(storageAccountType)
```

Comment:
can remove the accountType check in Line102?

Reply:
I would like to keep it, since there is also user specified account condition:
```
 	if len(storageAccount) > 0 {
  		accounts = append(accounts, accountWithLocation{Name: storageAccount})
```

Reply:
Yeah, meant to remove the logic for comparing storagetype but keep remaining.